schemas/com.roku/device_info/jsonschema/1-0-0

```diff
@@ -0,0 +1,159 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema with information about Roku device",
+  "self": {
+    "format": "jsonschema",
+    "name": "device_info",
+    "vendor": "com.roku",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "model": {
+      "type": "string",
+      "description": "Model name of the Roku device.",
+      "maxLength": 255
+    },
+    "modelDisplayName": {
+      "type": "string",
+      "description": "Model display name.",
+      "maxLength": 65535
+    },
+    "modelType": {
+      "type": "string",
+      "description": "Type of device.",
+      "maxLength": 255
+    },
+    "osVersion": {
+      "type": [
+        "string",
+        "null"
+      ],
```

Comment:
Why optional? Surely there's always an osVersion?

Reply:
Actually, not always because the SDK was only introduced in Roku OS 9.2 🤷‍♂️  There are some deprecated calls that give this information in a slightly different format but I didn't think it was worth the effort to add implement that – it requires some string parsing to find the OS version, cumbersome but possible to implement. What do you suggest?

Reply:
No, I'm fine if this is missing for older OSs 👍